src/docks/playlistdock.ui

```diff
@@ -59,14 +59,21 @@
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body&gt;
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
 &lt;p style=&quot; margin-top:12px; margin-bottom:16px; margin-left:-24px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Double-click&lt;/span&gt; a playlist item to open it in the player.&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:16px; margin-left:-24px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You can freely preview clips without necessarily adding them to the playlist or closing it.&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:16px; margin-left:-24px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To trim or adjust a playlist item &lt;span style=&quot; font-weight:600;&quot;&gt;Double-click&lt;/span&gt; to open it, make the changes, and click the &lt;span style=&quot; font-weight:600;&quot;&gt;Update&lt;/span&gt; icon.&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:16px; margin-left:-24px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Drag-n-drop&lt;/span&gt; to rearrange the items.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
+        <item>
+         <widget class="QPushButton" name="addFilesToPlaylistButton2">
+          <property name="text">
+           <string>Add files to playlist</string>
```

Comment:
The big button as wide as the panel is not acceptable. It should be added as an icon button.

For this we need to add a new icon - something along the lines of an import image. When dealing with icons in Shotcut, there are 3 icon themes. The dark and light fusion themes try to use something from the DefaultIcon collection, and the System theme uses one from the Oxygen collection. Our [Credits](https://shotcut.org/credits/) page has links to these. The DefaultIcon will directly work in the Fusion Light theme, and then we run an ImageMagick command to invert its color slightly less than pure white for the Fusion Dark theme. See `scripts/update-dark.sh` for the script I use. I can help you create that once we have decided. We generally use the 32x32 PNG icons from these. A lot of times the name alone cannot be used to locate the icon, and you need to browse the images. In some rare cases we need to create an icon from scratch or take an icon from one of the two collections and try to make it work under the other themes. I just checked and there is nothing obvious to use. In DefaultIcon, maybe `file-add.png` can be used, and in Oxygen I wonder if `insert-image.png` can be used.

Reply:
> ... the System theme uses one from the Oxygen collection. Our Credits page has links to these.

I went to the Oxygen webpage from the link provided in the Credits but I couldn't use it. I went to search an icon in the search box but the page didn't change.

> See scripts/update-dark.sh for the script I use. I can help you create that once we have decided.

Yes, I would appreciate this.

> In DefaultIcon, maybe MD-stream-video.png can be used

What about `file-add.png`?

Reply:
You can get Oxygen here: https://github.com/KDE/oxygen-icons

> What about file-add.png?

That is also an option if you do not mind something that looks like paper/document. I prefer something media-oriented.

> I can't seem to use ImageMagick to invert the DefaultIcon icon color.

Your results look fine to me. It can be difficult to see white on white.

Reply:
> I prefer something media-oriented.

Ok, I will use `MD-stream-video.png` then.